### PR TITLE
start: restyle with exos UI

### PR DESCRIPTION
**Pull request by printminion-co**

### CODE REVIEW

File: src/lib/components/chat/Greeting.svelte [hunk 1]
```diff
@@ -8,10 +8,7 @@
 </script>
 
 <div in:fade={{ duration: 100 }}>
-	<h1 class="text-2xl text-center sm:text-3xl font-bold line-clamp-1 text-ionos">
+	<h1 class="text-center exos-page-header sm:text-3xl line-clamp-1">
 		{$i18n.t('Welcome at IONOS GPT', { ns: 'ionos' })}
 	</h1>
-	<h4 class="text-base line-clamp-1">
-		{$i18n.t('Your first step into the world of AI! Privacy by Design!', { ns: 'ionos' })}
```
Comment (bromiesTM) [inline src/lib/components/chat/Greeting.svelte]: Removed texts could be removed from the translation file

File: src/routes/start/+page.svelte [hunk 2]
```diff
@@ -117,13 +117,6 @@
 		<SendMessageButton class="p-0 mx-2 flex w-8 group/button" disabled={model === null} />
 	</form>
 
-	<h1 class="my-2 text-center text-lg mb-0 mt-5">
-		{$i18n.t(
-			'... or let yourself inspire by these examples how our assistants can support your work',
```
Comment (bromiesTM) [inline src/routes/start/+page.svelte]: Same here. Remove from translation file

File: src/routes/start/+page.svelte [hunk 3]
```diff
@@ -114,7 +114,7 @@
 			className="w-full rounded-lg p-3 text-sm dark:text-gray-300 dark:bg-gray-850 outline-none resize-none h-full bg-transparent"
 		/>
 
-		<SendMessageButton class="p-0 mx-2 flex w-8 group/button" disabled={model === null} />
+		<SendMessageButton class="p-0 mx-2 flex w-8 group/button" disabled={model === null || prompt.length === 0} />
```
Comment (bromiesTM) [inline src/routes/start/+page.svelte]: We should strip whitespaces from length before to disable submit without actual text

File: src/lib/IONOS/components/ModelSelector.svelte [hunk 4]
```diff
@@ -36,12 +36,15 @@
...
+	<p class="exos-headline exos-headline--paragraph exos-headline--activating">{modelName}</p>
+</div>
 
 <style>
 	input {
 		display: none;
 	}
+	button {
+		margin-bottom: 0;
+	}
```
Comment (bromiesTM) [inline src/lib/IONOS/components/ModelSelector.svelte]: how about just `mb-0`

File: src/routes/start/+page.svelte [hunk 5]
```diff
@@ -96,25 +96,28 @@
 <div class="overflow-auto w-full h-full flex flex-col items-center">
 	<Greeting />
 
-	<h1 class="my-2 text-center text-lg mb-5 mt-5">
-		{$i18n.t('Choose the AI assistant that suits you.', { ns: 'ionos' })}
+	<h1 class="my-2 text-center text-lg mb-5 mt-32 exos-headline exos-headline--paragraph exos-headline--activating"
```
Comment (bromiesTM) [inline src/routes/start/+page.svelte:107]: this should be dark blue according to mockup

File: src/routes/start/+page.svelte [hunk 6]
```diff
@@ -96,25 +96,28 @@
...
 			bind:value={prompt}
+			rows=8
 			disabled={model === null}
 			placeholder={$i18n.t('Send a message to {{modelName}}', { ns: 'ionos', modelName })}
-			className="w-full rounded-lg p-3 text-sm dark:text-gray-300 dark:bg-gray-850 outline-none resize-none h-full bg-transparent"
+			className="exos-input-textarea"
 		/>
 
-		<SendMessageButton class="p-0 mx-2 flex w-8 group/button" disabled={model === null || prompt.length === 0} />
+		<SendMessageButton class="absolute p-0 mx-2 flex w-8 group/button right-2 bottom-2 {model === null || prompt.length === 0 ? 'hidden' : ''}" disabled={model === null || prompt.length === 0} />
```
Comment (bromiesTM) [inline src/routes/start/+page.svelte]: Length excluding whitespaces :)

File: src/routes/start/+page.svelte [hunk 7]
```diff
@@ -123,3 +126,9 @@
 
 	<Trailer />
 </div>
+
+<style>
+	#ai-prompt-input .-ai-icon {
+		color: var(--primary-shape-color);
+	}
```
Comment (bromiesTM) [inline src/routes/start/+page.svelte:144]: how about using `text-[var(--primary-shape-color)]` via tailwind

**Review by bromiesTM — APPROVED** [review]
Review ok but some minor amends todo

File: src/lib/IONOS/components/ModelSelector.svelte (hunk 4 above)
Comment (printminion-co) [reply src/lib/IONOS/components/ModelSelector.svelte]: !mb-0 works

**Pull request merged**